Windows/main.cpp

```diff
@@ -825,6 +825,24 @@ static void WinMainCleanup() {
 }
 
 int WINAPI WinMain(HINSTANCE _hInstance, HINSTANCE hPrevInstance, LPSTR szCmdLine, int iCmdShow) {
+	std::vector<std::wstring> wideArgs = GetWideCmdLine();
+
+	// Check for the Vulkan workaround before any serious init.
+	for (size_t i = 1; i < wideArgs.size(); ++i) {
+		if (wideArgs[i][0] == L'-') {
+			// This should only be called by DetectVulkanInExternalProcess().
+			if (wideArgs[i] == L"--vulkan-available-check") {
+				// Just call it, this way it will crash here if it doesn't work.
+				// (this is an external process.)
+				bool result = VulkanMayBeAvailable();
+
+				LogManager::Shutdown();
```

Comment:
Hm, not even starting up the log manager.  Might be safer not to call these things in case they start crashing in certain cases later when init never happened.

-[Unknown]

Reply:
hm, yeah, I got a little aggressive here - wanted the spawned process to be as pure a VK check as possible, and missed this path. I'll clean it up.